b'GLES3'

Backend terminated or disconnected.Fatal Python error: Segmentation fault

Current thread 0xb6f6dad0 (most recent call first):
  File "/usr/local/lib/python3.7/dist-packages/pi3d/Buffer.py", line 326 in draw
  File "/usr/local/lib/python3.7/dist-packages/pi3d/Shape.py", line 208 in draw
  File "/home/pi/Desktop/Github/Shaders-on-raspberry-pi4/New Test2/Paddy2.py", line 28 in <module>
  File "/usr/lib/python3/dist-packages/thonny/backend.py", line 1272 in _execute_prepared_user_code
  File "/usr/lib/python3/dist-packages/thonny/backend.py", line 1200 in wrapper
  File "/usr/lib/python3/dist-packages/thonny/backend.py", line 1213 in wrapper
  File "/usr/lib/python3/dist-packages/thonny/backend.py", line 1259 in execute_source
  File "/usr/lib/python3/dist-packages/thonny/backend.py", line 815 in _execute_source
  File "/usr/lib/python3/dist-packages/thonny/backend.py", line 801 in _execute_file
  File "/usr/lib/python3/dist-packages/thonny/backend.py", line 403 in _cmd_Run
  File "/usr/lib/python3/dist-packages/thonny/backend.py", line 204 in handle_command
  File "/usr/lib/python3/dist-packages/thonny/backend.py", line 146 in mainloop
  File "/usr/lib/python3/dist-packages/thonny/backend_launcher.py", line 87 in <module> Use 'Stop/Restart' to restart.
